AK/Format.cpp

```diff
@@ -885,7 +885,12 @@ ErrorOr<void> Formatter<T>::format(FormatBuilder& builder, T value)
         m_mode = Mode::String;
 
         Formatter<StringView> formatter { *this };
-        return formatter.format(builder, StringView { reinterpret_cast<char const*>(&value), 1 });
+
+        // convert value to single byte, important for big-endian because the LSB is the last byte.
+        VERIFY(value >= 0 && value <= 127);
```

Comment:
Let's move the `FIXME` comment from above down here then, since we already moved the `VERIFY`.